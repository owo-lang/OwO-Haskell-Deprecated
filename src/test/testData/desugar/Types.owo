module Types where
 A = Type
 B = Type0
 C = Type1
 D = TypeInf
