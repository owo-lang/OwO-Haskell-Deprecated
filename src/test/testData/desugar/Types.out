File type: owo
Top module name: .Types
 implementation of function A (20 2 2) (21 2 3)
 type
  meta variable _ (20 2 2) (21 2 3)
 value
  type universe Type (24 2 6) (28 2 10)
  level: ULevelLit 0
 implementation of function B (30 3 2) (31 3 3)
 type
  meta variable _ (30 3 2) (31 3 3)
 value
  type universe Type0 (34 3 6) (39 3 11)
  level: ULevelLit 0
 implementation of function C (41 4 2) (42 4 3)
 type
  meta variable _ (41 4 2) (42 4 3)
 value
  type universe Type1 (45 4 6) (50 4 11)
  level: ULevelLit 1
 implementation of function D (52 5 2) (53 5 3)
 type
  meta variable _ (52 5 2) (53 5 3)
 value
  type universe TypeInf (56 5 6) (63 5 13)
  level: ULevelMax
